.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - "6.9.1"
+  - "8.11.1"
```

Comment:
it was because with node v6.9.1 and the orginal `package.json`. Travis CI always fail (https://travis-ci.org/twosigma/git-meta/builds/656330522?utm_medium=notification&utm_source=github_status). I suspect it is because some public library was force updated with the same version. To make travis ci pass and to embrace newer NodeJs, I updated the node version